plugins/modules/files/json_file.py

```diff
@@ -0,0 +1,540 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, DEMAREST Maxime <maxime@indelog.fr>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import json
+import os
+
+from ansible_collections.community.general.plugins.module_utils.common.validation import check_type_dict_or_list
+from ansible_collections.community.general.plugins.module_utils.mh.module_helper_dest_file import (
+    DestFileModuleHelper,
+    ModuleHelperException,
+)
+from ansible_collections.community.general.plugins.module_utils.data_merge_utils import (
+    DataMergeUtils,
+    MergeType,
+    ListDiffType,
+)
+
+
+DOCUMENTATION = r'''
+---
+module: json_file
+short_description: Managage (add, remove, update) elements in a json file.
```

Comment:
```suggestion
short_description: Managage (add, remove, update) elements in a JSON file.
```
JSON is an abbreviation, so it should always be capitalized (when not referring to the module name or a variable/option of this name).